src/osfiber_windows.h

```diff
@@ -77,9 +79,12 @@ Allocator::unique_ptr<OSFiber> OSFiber::createFiber(
     size_t stackSize,
     const std::function<void()>& func) {
   auto out = allocator->make_unique<OSFiber>();
-  out->fiber = CreateFiber(stackSize, &OSFiber::run, out.get());
+  // Windows XP:  The FIBER_FLAG_FLOAT_SWITCH flag is not supported.
+  // CreateFiber -->> CreateFiberEx
+  // least stackSize 64KB
```

Comment:
According [to the docs](https://docs.microsoft.com/en-us/windows/win32/procthread/thread-stack-size):

> The operating system rounds up the specified size to the nearest multiple of the system's allocation granularity (typically 64 KB).

So maybe this should be:

`// stackSize is rounded up to the system's allocation granularity (typically 64 KB).`